src/main/g8/src/main/resources/application.conf

```diff
@@ -0,0 +1,10 @@
+twitter {
+  consumer {
```

Comment:
[Minor] We advise people to specify the tokens in env variables rather than in plain text in configuration files because it is more secure. 

..I'll merge this and quickly change it in another PR